src/contract/v_stable_swap_ctrt.js

```diff
@@ -498,6 +498,7 @@ export class StableSwapCtrt extends ctrt.Ctrt {
    */
   async getOrderOwner(orderId) {
     const rawVal = await this.queryDbKey(DBKey.forOrderOwner(orderId));
+    console.log(rawVal);
```

Comment:
`console.log` should be avoided in the codes. Please check & the rest part of the codes and remove `console.log` if any. @Meiyunoda